dcc-portal-ui/app/scripts/advanced/js/advanced.js

```diff
@@ -17,6 +17,8 @@
 
 'use strict';
 
+import ensureString from '../../common/js/ensure-input';
```

Comment:
Are you sure this works?

Does it not need to be `import { ensureString } ...`?